tsl/src/compression/compression.c

```diff
@@ -3123,7 +3174,10 @@ decompress_batches(RowDecompressor *decompressor, ScanKeyData *scankeys, int num
 			pos++;
 		}
 		if (skip_tuple)
+		{
+			num_filtered_rows++;
```

Comment:
The `skip_tuple` code path is because heap scans can't check for nulls, right? Would be good to add coverage for it (I thought we have it already, maybe it's just codecov being weird).